lametro/views.py

```diff
@@ -610,43 +610,55 @@ def __init__(self, *args, **kwargs):
 
         super(LAMetroCouncilmaticSearchForm, self).__init__(*args, **kwargs)
 
-    def clean_q(self):
-        q = self.cleaned_data['q']
+    def _full_text_search(self, sqs):
+        report_filter = SQ()
+        attachment_filter = SQ()
 
-        if q:
-            return ' AND '.join('({})'.format(term.strip()) for term in q.split('AND'))
-        else:
-            return ''
+        for token in self.cleaned_data['q'].split(' AND '):
+            report_filter &= SQ(text=Raw(token))
+            attachment_filter &= SQ(attachment_text=Raw(token))
```

Comment:
Most of this is reorganization. This is the main change: I revert back to issuing a `Raw` search for each term in the query, so double quotes are preserved. https://django-haystack.readthedocs.io/en/master/inputtypes.html#raw